File: frontend/extract.js
const fs = require('fs');
const html = fs.readFileSync('august_source.html', 'utf8');

// Find all SVGs
const svgs = html.match(/<svg[^>]*>[\s\S]*?<\/svg>/gi) || [];
console.log("--- FOUND SVGs ---");
svgs.forEach((svg, i) => {
    if (svg.length < 5000) { // skip massive inline svgs
        console.log(`\nSVG ${i}:`, svg);
    }
});

// Find all Hex colors
const colors = [...new Set(html.match(/#[0-9a-fA-F]{6}/g))];
console.log("\n--- FOUND COLORS ---");
console.log(colors);

// Look for image sources
const imgs = html.match(/<img[^>]+src="([^">]+)"/gi) || [];
console.log("\n--- FOUND IMAGES ---");
imgs.forEach((img, i) => console.log(`IMG ${i}:`, img));

File: frontend/extract.py
from bs4 import BeautifulSoup
import re

with open('august_source.html', 'r', encoding='utf-8') as f:
    soup = BeautifulSoup(f.read(), 'html.parser')

print("--- LOGOS / IMAGES IN HEADER ---")
for img in soup.find_all('img'):
    src = img.get('src')
    if src and ('logo' in src.lower() or 'header' in src.lower() or 'svg' in src.lower()):
        print(src)

for svg in soup.find_all('svg'):
    # print some identifying info
    print(svg.get('class'), str(svg)[:100])

print("\n--- COLORS ---")
colors = set(re.findall(r'#[0-9a-fA-F]{3,6}', str(soup)))
print("Colors found in HTML:", colors)
